docs/api_reference.rst

```diff
@@ -17,12 +17,11 @@ Modules
    :maxdepth: 1
 
    modules/spharpy.beamforming
-   modules/spharpy.indexing
    modules/spharpy.interpolate
    modules/spharpy.plot
    modules/spharpy.samplings
    modules/spharpy.spatial
-   modules/spharpy.spherical
+   modules/spharpy.special
```

Comment:
is it intended that spherical was removed and special stays?

Reply:
upps